administrator-manual/en/conf.py

```diff
@@ -115,19 +110,14 @@
        'Miscellaneous'),
     ]
     html_theme_options = {
-        'navbar_title': 'NethServer Enterprise',
-        'navbar_pagenav': True,
-        'navbar_sidebarrel': False,
-        'navbar_pagenav_name': 'Contents',
-        'navbar_class': "navbar",
-        'navbar_fixed_top': "false",
-        'source_link_position': "none",
-        'bootswatch_theme': "cerulean",
         'nosidebar': "1",
+        'collapse_navigation': True,
+        'navigation_depth': -1,
+        'logo_only': True,
+        'style_nav_header_background': '#343131',
     }
 
 elif tags.has('nscom'):
-    templates_path = ['nscom/_templates']
```

Comment:
Why was this line removed?

Reply:
There are no tenplates for nscom, so I proposed to remove it.
I'm ok even if we want keep it.